website/docs/plugin/framework/handling-data/paths.mdx

```diff
@@ -85,17 +85,18 @@ This pattern can be extended to as many calls as necessary. The different framew
 
 The following table shows the different [`path.Path` type](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/path#Path) methods associated with building paths for attribute implementations. Attribute types that cannot be traversed further are shown with N/A (not applicable).
 
-| Attribute Type            | Child Path Method |
-| ------------------------- | ----------------- |
-| `schema.BoolAttribute`    | N/A               |
-| `schema.Float64Attribute` | N/A               |
-| `schema.Int64Attribute`   | N/A               |
-| `schema.ListAttribute`    | `AtListIndex()`   |
-| `schema.MapAttribute`     | `AtMapKey()`      |
-| `schema.NumberAttribute`  | N/A               |
-| `schema.ObjectAttribute`  | `AtName()`        |
-| `schema.SetAttribute`     | `AtSetValue()`    |
-| `schema.StringAttribute`  | N/A               |
+| Attribute Type            | Child Path Method                                                                                      |
+|---------------------------|--------------------------------------------------------------------------------------------------------|
+| `schema.BoolAttribute`    | N/A                                                                                                    |
+| `schema.DynamicAttribute` | Based on underlying value, see [Building Dynamic Attribute Paths](#building-dynamic-attribute-paths)   |
```

Comment:
Just a reminder note to adjust this if/when the path handling is removed

Reply:
Adjusted in 2f9d877 ⚡